### PR TITLE
01 - Implemented start index.html page, added css styles

### PR DESCRIPTION
**Pull request by izabutana**
Implemented start index.html page, added css styles. 

**Start page:**
image

**Issue** - #1

### CODE REVIEW

**Review by vladkuzmich — APPROVED** [review]

File: src/styles.css [hunk 1]
```diff
@@ -1 +1,24 @@
-/* You can add global styles to this file, and also import other style files */
+body{
+    background-color:#e6e6fa;
+   } 
+  .navbar {
+    border: none;
```
Comment (pryabov) [inline src/styles.css]: this is a default value. Should be removed

**Review by pryabov — CHANGES_REQUESTED** [review]
Could you change formatting of CSS file?

image

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,20 +1,5 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-  <h1>
-    Welcome to {{ title }}!
-  </h1>
-  <img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
+<div class="navbar">
+        <div class="navbar-brand">Mastery Web Chat</div>
```
Comment (pryabov) [inline src/app/app.component.html]: usually, image as a logo is placed in the first position on NavBar.

**Review by pryabov — CHANGES_REQUESTED** [review]
Also, pls, setup minimum page width to prevent something like this:

image

**Review by pryabov — CHANGES_REQUESTED** [review]
- NavBar should be a separate component
- Alos make sure that you write css code in proper files (right now it looks like better to move CSS to a component level, not global `Style.css`)

File: src/styles.css [hunk 3]
```diff
@@ -1,26 +1,5 @@
 body{
```
Comment (pryabov) [inline src/styles.css]: pls reformat this css file

**Review by pryabov — APPROVED** [review]

**Pull request merged**